.github/workflows/test-python.yml

```diff
@@ -77,3 +72,31 @@ jobs:
           net start MongoDB
       - run: just install
       - run: just test
+
+  build-min:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        persist-credentials: false
+        fetch-depth: 0
+    - name: Install uv
+      uses: astral-sh/setup-uv@v5
+      with:
+        enable-cache: true
+        python-version: "3.9"
+    - uses: extractions/setup-just@v2
+    - name: Install uv
+      uses: astral-sh/setup-uv@v5
+      with:
+        enable-cache: true
+        python-version: "3.9"
```

Comment:
Similar to comment about ${MONGODB_VERSION} => 7.0,
MIN_PYTHON_VERSION: "3.9", MONGODB_VERSION: "4.0" seem like a good idea. What am I missing?

Reply:
Updated